com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1217,173 +1380,47 @@ private List<TaskResult> doBuild(IProgress monitor, String... commands) throws I
                 case "build": {
                     ExtenderUtil.checkProjectForDuplicates(this); // Throws if there are duplicate files in the project (i.e. library and local files conflict)
 
-                    final Boolean withSymbols = this.hasOption("with-symbols");
                     final String[] platforms = getPlatformStrings();
+                    Future<Void> remoteBuildFuture = null;
```

Comment:
The Future will hold the result of the remote build (which is now running in a separate thread)